Adapter/PlentymarketsAdapter/ResponseParser/Product/Variation/VariationResponseParser.php

```diff
@@ -451,4 +451,20 @@ private function getVariationPropertyTranslations(array $names)
 
         return $translations;
     }
+
+    /**
+     * @param array $variation
+     *
+     * @return float
+     */
+    private function getVariationWeight(array $variation)
+    {
+        if ($variation['weightNetG'] > 0) {
+            $weight = $variation['weightNetG'];
+        } else {
+            $weight = $variation['weightG'];
```

Comment:
if this value is in KG, shouldnt we devide by 1000 before processing it further?

Reply:
Siehe Zeile 468…